templates/no-res-name.html

```diff
@@ -10,25 +10,30 @@ <h4 class="alert-heading">No results</h4>
       Remember not to include extra characters like "Sr" or middle initials.</p>
     <p>
       If you're sure you got it right, please reach out to them and find out
-      about their voting plans. They can request an absentee ballot online,
-      using <a href="https://ballotrequest.sos.ga.gov/" target="_blank" rel="noopener noreferrer">ballotrequest.sos.ga.gov</a>.
+      about their voting plans.
+      They can <a href="https://www.voteamerica.com/where-to-vote/" target="_blank" rel="noopener noreferrer">find their early voting location here.</a>
     </p>
-    <p class="my-0">
-      It can take a few days for results to show up here. If all else fails,
-      they may want to
-      <a href="https://www.voteamerica.com/local-election-offices/GA/" target="_blank" rel="noopener noreferrer">contact
-      their Local Elections Office</a>.
+    <p class="mb-0">
+      It can take a few days for results to show up here.
+      If they mailed their ballot more than 7 days ago, or voted in person more than 3 business days ago,
+      they may want to <a href="https://www.voteamerica.com/local-election-offices/GA/" target="_blank" rel="noopener noreferrer">contact their Local
+      Elections Office</a>.
     </p>
   </div>
 
   <div class="alert alert-info" role="alert">
-    All of this data comes from the <a href="https://elections.sos.ga.gov/Elections/voterabsenteefile.do  target="_blank" rel="noopener noreferrer"">publicly
-    available files</a> published by the Georgia Secretary of State. If
-    you are wanting to track your own ballot status, consider utilizing
-    <a href="https://georgia.ballottrax.net/voter/" target="_blank" rel="noopener noreferrer">Georgia's official
-    site for this purpose</a>.
+    <p>
+      All of this data comes from the <a href="https://elections.sos.ga.gov/Elections/voterabsenteefile.do" target="_blank" rel="noopener noreferrer"">publicly
+      available files</a> published by the Georgia Secretary of State. If
+      you want to track your own ballot status, use
+      <a href="https://www.mvp.sos.ga.gov/MVP/mvp.do" target="_blank" rel="noopener noreferrer">Georgia's official
+      site for this purpose</a>.
+    </p>
+    <p>
```

Comment:
```suggestion
    <p class="mb-0">
```

to match yellow box above, or remove them both